app/helpers/country_selection_helper.rb

```diff
@@ -10,7 +10,9 @@ def set_country_selection_options
     assessment_structure = JSON.load File.open './app/fixtures/assessments.json'
     assessments = Assessment.pluck(:country, :assessment_type)
     selectables =
-      assessments.reduce({}) do |acc, (country, assessment_type)|
+      assessments.reduce(
+        { '-- Select One --' => [] }
```

Comment:
why not use the `include_blank` option on `select_tag` instead?

Reply:
I didn't know about it before. But, it's also turning out to be challenging to manage with the menu where we replace all of the options in the menu in response to an action in a different menu.

Reply:
Turns out that since we have a dynamic menu, it's actually much easier to manage the state if the placeholder option is a full option presented in the Javascript and not just in the template code.

In particular, when I change the selected country, I would have to manually re-add the option in the Javascript before populating the rest of the assessments.